SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -21,7 +21,14 @@ void USpatialWorkerConnection::SetConnection(Worker_Connection* WorkerConnection
 		if (OpsProcessingThread == nullptr)
 		{
 			bool bCanWake = SpatialGDKSettings->bWorkerFlushAfterOutgoingNetworkOp;
-			ThreadWaitCondition.Emplace(bCanWake, 1.0f / GetDefault<USpatialGDKSettings>()->OpsUpdateRate);
+			float WaitTimeS = 1.0f / (GetDefault<USpatialGDKSettings>()->OpsUpdateRate);
+			int32 WaitTimeMs = static_cast<int32>(FTimespan::FromSeconds(WaitTimeS).GetTotalMilliseconds());
```

Comment:
Is it actually possible for (1/x) to be == 0 ?

Reply:
Nah but its the integer conversion which breaks it - the sleep functions take milliseconds in integer form

Reply:
Ahh gotcha